test/make.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This function checks to make sure that every
+# shebang has a '- e' flag, which causes it
+# to exit on error
+function check_bash() {
+find . -name "*.sh" | while IFS= read -d '' -r file;
+do
+  if [[ "$file" != *"bash -e"* ]];
+  then
+    echo "$file is missing shebang with -e";
+    exit 1;
+  fi;
+done;
+}
+
+# This function makes sure that the required files for
+# releasing to OSS are present
+function basefiles() {
+  echo "Checking for required files"
+  test -f LICENSE || echo "Missing LICENSE"
+  test -f README.md || echo "Missing README.md"
+}
+
+# This function runs the hadolint linter on
+# every file named 'Dockerfile'
+function docker() {
+  echo "Running hadolint on Dockerfiles"
+  find . -name "Dockerfile" -exec hadolint {} \;
+}
+
+# This function runs 'terraform validate' against all
+# files ending in '.tf'
+function check_terraform() {
+  echo "Running terraform validate"
+  #shellcheck disable=SC2156
+  find . -name "*.tf" -exec bash -c 'terraform validate --check-variables=false $(dirname "{}")' \;
```

Comment:
This looks like it's missing a few `-not path`s from the module template. Does `make` succeed?